### ISSUE TITLE
Python API is under-documented

### ISSUE THREAD

**Issue by elistevens** [nandoflorestan/kajiki#29]
In specific, loader.py doesn't have `FileLoader` mentioned at all, nor do the docs spell out the accepted filename extensions for `PackageLoader`.

Also, the online docs seem to be from v0.4.

Are there any initiatives to improve the situation?

**Comment by amol-** [comment]
Yes, that has been pretty much left behind because the primary way to use Kajiki is through TurboGears which provide a customised loader and sets things up for you. So documentation on custom usage of kajiki has left little behind.

Also make sure you look at http://kajiki.readthedocs.io/ that the last version of documentation

**Comment by elistevens** [comment]
Thanks for the link.

The github README, the pypi page, and even the readthedocs page itself all link to the older pythonhosted docs, FYI.  :)

**Comment by amol-** [comment]
Fixed! They were all including `README.rst` which had the wrong link
